src/main/msp/msp_serial.c

```diff
@@ -215,7 +296,11 @@ void mspSerialProcess(mspEvaluateNonMspData_e evaluateNonMspData, mspProcessComm
         if (!mspPort->port) {
             continue;
         }
+
+        mspPortBeingProcessed = mspPort;
```

Comment:
this is so that requests to stream apply only to the port that received the command.  i.e. in the case of the OSD slave we don't spam the USB port AND UART1 with MSP_ANALOG replies, only UART1.